src/amp-shadow.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The entry point for AMP Runtime (v0.js) when AMP Runtime may support
+ * multiple AMP Docs in Shadow DOM.
+ */
+
+import './polyfills';
```

Comment:
No. The idea is that this is the special version of `amp.js` (`v0.js`) that will be pulled by shell script/document. So, only this file is loaded, not `amp.js`.